app/components/works/work_form_component.html.erb

```diff
@@ -1,8 +1,10 @@
 <div id="editor">
   <h1 class="row">Deposit your work</h1>
   <%= form_with model: work_form, url: [work_form.model.collection, work_form],
-      local: true,
```

Comment:
Removing `local: true` converts this to a "remote form" (https://guides.rubyonrails.org/working_with_javascript_in_rails.html).